### PR TITLE
docs: add release v1.28 boilerplate for blogpost

### PR DESCRIPTION
**Pull request by KollaAdithya**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Comment by KollaAdithya** [comment]
Please let me know if i miss any features that needs to be added to blogpost boilerplate.

File: site/content/blogs/release-v128.en.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.28: Static Site Service Type and more!
+
+Posted On: May 02, 2023
+
+The AWS Copilot core team is announcing the Copilot v1.28 release.
+Special thanks to [@interu](https://github.com/interu), [@0xO0O0](https://github.com/0xO0O0),who contributed to this release.
```
Comment (paragbhingre) [inline site/content/blogs/release-v128.en.md]:
```suggestion
Special thanks to [@interu](https://github.com/interu), [@0xO0O0](https://github.com/0xO0O0), who contributed to this release.
```

File: site/content/blogs/release-v128.en.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: 'AWS Copilot v1.28: Static Site Service Type and more!'
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md]:
```suggestion
title: 'AWS Copilot v1.28: Static Site service type and more!'
```

File: site/content/blogs/release-v128.en.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+twitter_title: 'AWS Copilot v1.28'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.28: Static Site Service Type and more!
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md]:
```suggestion
# AWS Copilot v1.28: Static Site service type and more!
```

File: site/content/blogs/release-v128.en.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Posted On: May 02, 2023
+
+The AWS Copilot core team is announcing the Copilot v1.28 release.
+Special thanks to [@interu](https://github.com/interu), [@0xO0O0](https://github.com/0xO0O0), who contributed to this release.
+Our public [сommunity сhat](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im) is growing and has over 400 people online and over 2.8k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.28 brings several new features and improvements:
+
+- **Static Site Service Type**: You can now deploy and host static website in AWS S3 Bucket. [See detailed section](#static-site-service-type).
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md]:
```suggestion
- **Static Site service type**: You can now deploy and host static websites with AWS S3. [See detailed section](#static-site-service-type).
```

File: site/content/blogs/release-v128.en.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- **Parallel Container Images Build**: You can now build main container and sidecar container images in parallel. With parallel build, you can reduce the overall time it takes build and push container images to AWS ECR repository.
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md]:
```suggestion
- **Parallel Container Images Build**: You can now build main container and sidecar container images in parallel. With parallel build, you can reduce the overall time it takes build and push container images to AWS ECR.
```

**Review by bvtujo — APPROVED** [review]

File: site/content/blogs/release-v128.en.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+The AWS Copilot core team is announcing the Copilot v1.28 release.
+Special thanks to [@interu](https://github.com/interu), [@0xO0O0](https://github.com/0xO0O0), who contributed to this release.
+Our public [сommunity сhat](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im) is growing and has over 400 people online and over 2.8k stars on [GitHub](http://github.com/aws/copilot-cli/).
+Thanks to every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.28 brings several new features and improvements:
+
+- **Static Site service type**: You can now deploy and host static websites with AWS S3. [See detailed section](#Static-Site-service-type).
+- **Config Multiple Container Ports through `--port` flag**:`Copilot init` and `Copilot svc init` now allows you to configure multiple ports for Load Balanced Service and Backend Service. [See detailed section](#Config-Multiple-Container-Ports-through---port-flag).
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md]:
```suggestion
- **Config Multiple Container Ports through `--port` flag**:`copilot init` and `copilot svc init` now allow you to configure multiple ports for Load Balanced Service and Backend Service. [See detailed section](#Config-Multiple-Container-Ports-through---port-flag).
```

File: site/content/blogs/release-v128.en.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+- **Container Images Parallel Build**: You can now build main container and sidecar container images in parallel. With parallel build, you can reduce the overall time it takes build and push container images to AWS ECR.
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md:23]: ECR is actually an Amazon product, not AWS
```suggestion
- **Container Images Parallel Build**: You can now build main container and sidecar container images in parallel. With parallel build, you can reduce the overall time it takes build and push container images to Amazon ECR.
```

File: site/content/blogs/release-v128.en.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+    See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## Static Site service type
+
+### Static Site Upload Experience
+
+### Integrate With CloudFront
+
+## Config Multiple Container Ports through `--port` flag
```
Comment (huanjani) [inline site/content/blogs/release-v128.en.md]:
```suggestion
## Config multiple container ports with the `--port` flag
```

**Review by huanjani — APPROVED** [review]
Looks good! Just a few nits, so added the DNM label; feel free to remove when ready!

**Pull request merged**